panther_bringup/README.md

```diff
@@ -6,18 +6,18 @@ The package contains the default configuration and launch files necessary to sta
-- `system_status_node` [*panther_manager/system_status_node.py*]: publishes information about the status of a Build-in Computer.
-- `welcome_msg_node` [*panther_bringup/welcome_msg_node.py*]: user-friendly welcome message with basic information about the given robot.
+- `battery_node` [<= 1.06 *[panther_battery/roboteq_republisher_node.py](../panther_battery/src/roboteq_republisher_node.py)*, >= 1.2 *[panther_battery/adc_node.py](../panther_battery/src/adc_node.py)*]: node responsible for monitoring and publishing the internal battery state of the Husarion Panther robot. For more information, refer to: [panther_battery](../panther_battery/README.md).
+- `driver_node` [*[panther_driver/driver_node.py](../panther_driver/src/driver_node.py)*]: node responsible for communication with motor controllers and computing the inverse and forward kinematics of a robot. For more information, refer to: [panther_driver](../panther_driver/README.md).
+- `ekf_node` [*[robot_localization/ekf_localization_node](https://github.com/cra-ros-pkg/robot_localization/blob/noetic-devel/src/ekf_localization_node.cpp)*]: Extended Kalman Filter node for more accurate odomtery. For more information, refer to: [robot_localization](https://github.com/cra-ros-pkg/robot_localization/tree/noetic-devel). The default configuration is stored in [ekf_config.yaml](./config/ekf_config.yaml).
+- `imu_filter_node` [*[imu_filter_madgwick/imu_filter_node](https://github.com/CCNYRoboticsLab/imu_tools/blob/noetic/imu_filter_madgwick/src/imu_filter_node.cpp)*]: node responsible for filtering and fusing raw data from IMU.
+- `lights_controller_node` [*[panther_lights/controller_node.py](../panther_lights/src/controller_node.py)*]: node responsible for processing animations and publishing frames to be displayed on the Husarion Panther robot LED panels. For more information, refer to: [panther_lights](../panther_lights/README.md).
+- `lights_driver_node` [*[panther_lights/driver_node](../panther_lights/src/driver_node.cpp)*]: node responsible for displaying frames on the Husarion Panther robot LED panels. For more information, refer to: [panther_lights](../panther_lights/README.md).
+- `manager_bt_node` [*[panther_manager/manager_bt_node](../panther_manager/src/manager_bt_node.cpp)*]: node responsible for managing the Husarion Panther robot. Incorporates a system for managing displayed LED panel animations, safety features, and software shutdown of components. For more information, refer to: [panther_manager](../panther_manager/README.md).
+- `phidgets_spatial_node` [*nodelet/nodelet*, type: **[phidgets_spatial/PhidgetsSpatialNodelet](https://github.com/ros-drivers/phidgets_drivers/blob/noetic/phidgets_spatial/src/phidgets_spatial_nodelet.cpp)**] - Phidget Spatial IMU ROS driver.
```

Comment:
IMO bold for this link looks a bit weird compared to other links. Maybe also itylic?